internal/holos/types.go

```diff
@@ -25,3 +26,23 @@ func (i *StringSlice) Set(value string) error {
 	}
 	return nil
 }
+
+type feature string
+
+const BuildFeature = feature("BUILD")
+const ServerFeature = feature("SERVER")
+const PreflightFeature = feature("PREFLIGHT")
+const GenerateComponentFeature = feature("GENERATE_COMPONENT")
+const SecretsFeature = feature("SECRETS")
```

Comment:
@natemccurdy FYI, I hid the preflight command behind the `HOLOS_FEATURE_PREFLIGHT` env var flag.  Setting this var, and others, will unhide the old subcommands.